### DISCUSSION TITLE
Doesn't create control for HX750i, help!

### DISCUSSION THREAD

**Discussion by OutsideOctaves** [Rem0o/FanControl.Releases#885] [Q&A]
There are two sensors it's seeing with the HX750i PSU, but only creates one for the "CASE" as it's calling it (CASE, and VRM are the two it sees) even though those are part of the PSU.  However, for some odd reason it's not creating a controller for it.  Is there a way to manually create a controller?  Rescanning does nothing.

I checked the box to search for the Corsair PSU

Any help here would be appreciated.  I'm new to this software, Jayz2Cents sent me ;) heheh.

**Comment by Rem0o** [comment]
What you see is what you get basically.

**Reply by OutsideOctaves** [reply]
So that's a No?  Which means that Corsair's icue is the only thing that can control it?  Thank you though.  Hopefully sometime in the future it will control stuff like this but thanks for this massively powerful piece of software.

**Reply by Rem0o** [reply]
Corsair PSU support is implemented here. Support might be limited. You can take a look:
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/master/LibreHardwareMonitorLib/Hardware/Psu/Corsair/CorsairPsu.cs

**Comment by OutsideOctaves** [comment]
Oh hey!  Ok, I'll look into how to impliment this.  Thank you :)
